test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlSerializerOutputFormatterTest.cs

```diff
@@ -68,6 +70,59 @@ public static IEnumerable<object[]> BasicTypeValues
             XmlAssert.Equal(expectedOutput, content);
         }
 
+        public static IEnumerable<object[]> CanIndentOutputConditionallyData
+        {
+            get
+            {
+                var obj = new DummyClass { SampleInt = 10 };
+
+                yield return new object[] { true, obj, "<DummyClass xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" " +
+                    "xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <SampleInt>10</SampleInt>\r\n</DummyClass>" };
```

Comment:
This looks like it needs to use `Environment.NewLine` because it causes a test failure on non-Windows:

```
Expected: ···org/2001/XMLSchema">\r\n  <SampleInt>10</SampleInt>\r\n</DummyCla···
Actual:   ···org/2001/XMLSchema">\n  <SampleInt>10</SampleInt>\n</DummyClass···
```